src/pages/orgs/[organizationId]/forks/[forkId].tsx

```diff
@@ -198,6 +202,20 @@ const SingleFork = (
                   <Link href={mirror.html_url} target="_blank">
                     {mirror.name}
                   </Link>
+                  <Box>
+                    <Button
+                      variant="danger"
+                      onClick={() => {
+                        deleteMirror({
+                          orgId: orgData.id.toString(),
+                          orgName: orgData.login,
+                          mirrorName: mirror.name,
+                        })
+                      }}
+                    >
+                      Delete
+                    </Button>
+                  </Box>
```

Comment:
Follow up PR: let's add a confirm because I know I will fat finger the delete button 😆